dockerfiles/ubuntu/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM concourse/golang-builder as builder
+COPY . /go/src/github.com/concourse/pool-resource
+ENV CGO_ENABLED 0
+ENV GOPATH /go/src/github.com/concourse/pool-resource/Godeps/_workspace:${GOPATH}
+ENV PATH /go/src/github.com/concourse/pool-resource/Godeps/_workspace/bin:${PATH}
+RUN go build -o /assets/out github.com/concourse/pool-resource/cmd/out
+RUN set -e; for pkg in $(go list ./...); do \
+		go test -o "/tests/$(basename $pkg).test" -c $pkg; \
+	done
+
+FROM concourse/git-resource AS resource
```

Comment:
Is this right? 🤔 I mean, do we intend to have this dependency on `git-resource`? And, if so, should we specify the `ubuntu` tag here?

Thanks!

Reply:
Adding to the comment above - should we also tag the dependency in `alpine` to use `:alpine` too?

Thanks!

Reply:
Good point, updated the PR